scripts/start.py

```diff
@@ -134,6 +141,7 @@ def main(args=None):
     with contextlib.ExitStack() as stack, alert_on_exit():
         # ExitStack unwinds in reverse-order, so this will be the final action.
         stack.callback(notify_about_successful_shutdown)
+        stack.callback(call_extend_index_yaml)
```

Comment:
cannot callback directly but will be fine with a function

Reply:
Great!